GREEDY/View/MainScreen.cs

```diff
@@ -1,87 +1,50 @@
-                string singleReceiptPath = ConfigurationManager.AppSettings["singleReceiptPath"];
-                var receipt = await new OCRController().UseOCRAsync(imageForOCR.FileName);
-                textResult.Text = string.Empty;
-                foreach (var line in receipt.LinesOfText)
-                {
-                    textResult.Text += line;
-                }
-                new CreatePathForDataController().CreateAFolder(receiptsFolder);
-                new WritingToFileController().WriteToFile(singleReceiptPath, receipt);
-
-                //writing raw data from receipt to items datatable 
-                RawDataFormatController rawDataFormatController = new RawDataFormatController(receipt);
-                ItemsList.DataSource = rawDataFormatController.GetDataTable();
-
-                //reading list of all items and adding new items from receipt list
-                string itemsListsFolder = ConfigurationManager.AppSettings["itemsListsFolder"];
-                string singleItemsListPath = ConfigurationManager.AppSettings["singleItemsListPath"];
-                new CreatePathForDataController().CreateAFolder(itemsListsFolder);
-                DataConvertionController dataConvertionController = 
-                    new DataConvertionController();
-
-                //reading all items from an xml file
-                List<Item> listOfAllItems = 
-                    dataConvertionController.XmlToList(singleItemsListPath);
-
-                //retrieving new receipts data and displaying it to DataGridView
-                DataTable newReceipDataTable = 
-                    new RawDataFormatController(receipt).GetDataTable();
-                ItemsList.DataSource = newReceipDataTable;
-
-                //converting new receipts datatable to List<Item>
-                List<Item> newReceipItems = 
-                    dataConvertionController.DataTableToList(newReceipDataTable);
-
-                //adding new items to the listOfAllItems
-                listOfAllItems.AddRange(newReceipItems);
-
-                //writing the listOfAllItems to the xml file
-                dataConvertionController.ListToXml(listOfAllItems, singleItemsListPath);
-
-                Application.UseWaitCursor = false;
-                btnOCR.Enabled = true;
-            }
-            GC.Collect();
+            var image = _photoImageGetter.GetImage();
+            Application.UseWaitCursor = true;
+            InserFile_Button.Enabled = false;
+            var processedReceipt = _receiptService.ProcessReceiptImage(image);
+            ItemList.DataSource = processedReceipt;
+            Application.UseWaitCursor = false;
+            InserFile_Button.Enabled = true;
         }
 
-        private void textResult_TextChanged(object sender, EventArgs e)
-        {
+        private void XMLdataGridView_CellContentClick(object sender, DataGridViewCellEventArgs e) { }
```

Comment:
Why are there empty event handlers?

Reply:
Norhing, but we have to have it because this pats is in our windows form application. If you delete it you will lost visual demostration of windos form

Reply:
You have to either remove them through properties, or remove manually remove the methods from this file and InitializeComponent method

Reply:
this delete item (table), but i need them for data display